src/tests/pools/D3MCompoundPool.t.sol

```diff
@@ -218,35 +217,39 @@ contract D3MCompoundPoolTest is D3MPoolBaseTest {
         assertEq(pool.redeemable(), address(cDai));
     }
 
-    function test_transfer_cdai() public {
-        pool.file("hub", address(this));
+    function test_exit_cdai() public {
```

Comment:
We might want a test that does 2 exits with an elapsing blocks (time in aave) period between them.
We want to make sure all dai and cdai were handed out.

Reply:
Added new tests.